app/controllers/users/registrations_controller.rb

```diff
@@ -1,11 +1,28 @@
-# frozen_string_literal: true
-
 class Users::RegistrationsController < Devise::RegistrationsController
 
+  def create
+    if email_matches_whitelist
+      super
```

Comment:
This doesn't feel very expressive.

Wrapping the call in a more expressive method (e.g. `do_default_devise_behaviour`) has some awkwardness with `super`, and also feels a little overkill?